index.html

```diff
@@ -224,6 +224,34 @@
       </features>
 
 <script>
+  // Callback for GitHub API; fills out version tags with the latest version
+  updateVersions = function(response) {
+    // Return the largest array, considered lexicographically
+    maxArray = function(a, b) {
+      for (var i = 0; i < Math.min(a.length, b.length); i++) {
+        if (a[i] < b[i]) {
+          return b
+        } else if (a[i] > b[i]) {
+          return a
+        }
+      }
+      return (a.length < b.length) ? b : a
+    }
+
+    var latestTag = (response.data
+        .filter(function(tag) { return /^v\d+(\.\d+)+$/.test(tag.name) })
```

Comment:
Is it ok these days to use these functional things? They haven't been around for that long yet, right? How does this fail on e.g. IE8?